code/weapon/weapons.cpp

```diff
@@ -1982,6 +1982,10 @@ int parse_weapon(int subtype, bool replace, const char *filename)
 		if ( required_string("+Bitmap:") ) {
 			stuff_string(fname, F_NAME, NAME_LENGTH);
 			generic_bitmap_init(&ti->texture, fname);
+
+			if (ti->texture.bitmap_id == -1) {
+				Warning(LOCATION, "Trail bitmap %s could not be loaded. Trail will not be rendered.", fname);
+			}
```

Comment:
Uh, is this in the right place? `generic_bitmap_init()` always sets `bitmap_id` to -1, so I assume this warning should be after the load is actually attempted.

Reply:
oops, changed